data/pathfinder/paizo/roleplaying_game/ultimate_intrigue/ui_equip_magic_items.lst

```diff
@@ -21,7 +21,7 @@ Costume Bureau							TYPE:Magic.Wondrous.GreaterMinor				COST:7000		WT:175		SOUR
 Courier's Secure Pouch						TYPE:Magic.Wondrous.LesserMinor				COST:1600		WT:2			SOURCEPAGE:p.245
 Communique Ring							TYPE:Magic.Ring.GreaterMinor					COST:6000					SOURCEPAGE:p.241
 Deadened Shadows Cloak						TYPE:Magic.Wondrous.Cloak.GreaterMajor			COST:63250		WT:2			SOURCEPAGE:p.246
-Deck of Doors							TYPE:Magic.Wondrous.LesserMedium				COST:12150					SOURCEPAGE:p.246
+Deck of Doors							TYPE:Magic.Wondrous.LesserMedium				COST:12150			EQMOD:Special Ability ~ Charged Item / 54 Maximum|CHARGES[54]	SOURCEPAGE:p.246
```

Comment:
Unconstructed Reference: EquipmentModifier Special Ability ~ Charged Item / 54 Maximum

Did you miss updating another file?